src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -317,6 +317,46 @@ public function symlink($originDir, $targetDir, $copyOnWindows = false)
     }
 
     /**
+     * Return the target of a link.
+     *
+     * @param string $path        A link path.
+     * @param bool   $finalTarget Whether or not to recursively follow links
+     *                            until a final target is reached.
+     *
+     * @return string Return the resolved link.
+     *
+     * @throws IOException When the link does not exist or is not readable.
+     */
+    public function readlink($path, $finalTarget = false)
```

Comment:
Or just keep realpath, which is very useful. I'm not sure readlink is that useful anyway.